packages/features/ee/workflows/pages/index.tsx

```diff
@@ -99,6 +103,10 @@ function WorkflowsPage({ filteredList }: PageProps) {
                     disableMobileButton={true}
                     onlyShowWithTeams={true}
                     includeOrg={true}
+                    withPermission={{
```

Comment:
The user should still be able to create individual workflows even when the team doesn't allow it